smart_contracts.rst

```diff
@@ -721,7 +721,9 @@ Protocol Values Constraints
     (2 TN) Wk <= Wt, time k < time t
     (3 R) Tk <= Tt, time k < time t
 
-Channel deposits, channel withdraws, off-chain transferred amounts are all monotonically increasing in time. The ``TokenNetwork`` contract must enforce this for deposits and withdraws. The Raiden client must enforce this for the off-chain transferred amounts, contained in the balance proofs.
+Channel deposits, channel withdraws, off-chain transferred amounts are all monotonically increasing in time.
```

Comment:
what do you mean by "monotonically increasing **in time**"?

Reply:
Yes, the time mention is redundant - I changed it to `monotonically increasing` now.